.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,35 +1,25 @@
-## PR Checklist
-Please check if your PR fulfills the following requirements:
-
-- [ ] Tests for the changes have been added (for bug fixes / features)
-- [ ] Docs have been added / updated (for bug fixes / features)
+## PR Description 
```

Comment:
Technically you can delete this line as the git commit text will be prepended to this.

Reply:
Reworked.